packages/widgetbook/lib/src/widgets/sorters/sort_button.dart

```diff
@@ -0,0 +1,63 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import 'package:widgetbook/src/models/models.dart';
+import 'package:widgetbook/src/navigation/organizer_provider.dart';
+import 'package:widgetbook/src/workbench/workbench_button.dart';
+
+class SortButton extends StatelessWidget {
+  const SortButton({
+    Key? key,
+    required this.activeSorting,
+    this.height = 17.0,
+  }) : super(key: key);
+
+  final Sorting? activeSorting;
+  final double height;
+
+  String _getTooltip() {
+    if (activeSorting == null) {
+      return 'Sort by name ascending';
+    }
+
+    switch (activeSorting!) {
+      case Sorting.asc:
+        return 'Sort by name descending';
+      case Sorting.desc:
+        return 'Sort by name ascending';
```

Comment:
Hey @Xazin, thanks for checking this out! 

It could be indeed confusing, but idea is to display the next action on that button. For example, if the current state is `Sorting.asc`, the button displays `Z->A` as this is exactly what happens when you press it.

Let me, please, know if I should change this.